gnomad/utils/annotations.py

```diff
@@ -2,6 +2,7 @@
 
 import logging
 from typing import Any, Dict, List, Optional, Set, Tuple, Union
+import json
```

Comment:
use isort to help format imports:
"import json" should come before "import logging"